packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -171,6 +171,12 @@ class MouseTracker {
     return trackedAnnotation;
   }
 
+  /// Checks if the given [MouseTrackerAnnotation] is attached to this
+  /// [MouseTracker].
+  bool isAnnotationAttached(MouseTrackerAnnotation annotation) {
```

Comment:
Make this `@visibleForTesting` too: it's not generally useful, so we don't want to add to the API surface unnecessarily.